cloud-controller-manager/do/loadbalancers.go

```diff
@@ -1388,6 +1393,17 @@ func getType(service *v1.Service) (string, error) {
 	return name, nil
 }
 
+func getNetwork(service *v1.Service) (string, error) {
+	network, ok := service.Annotations[annDONetwork]
+	if !ok || network == "" {
```

Comment:
nit: could be simplified to

```golang
if network == "" {
```

since the default value matches the absent one.

(I realize we have this redundant in a few spots in CCM.)